src/main/java/com/ft/methodearticlemapper/transformation/MappedDataBodyProcessingContext.java

```diff
@@ -0,0 +1,29 @@
+package com.ft.methodearticlemapper.transformation;
+
+import com.ft.bodyprocessing.DefaultTransactionIdBodyProcessingContext;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.stream.Collectors;
+
+public class MappedDataBodyProcessingContext extends DefaultTransactionIdBodyProcessingContext {
```

Comment:
should we also have an interface which exposes the `get` method so that EventHandler(s) don't have to cast the `BodyProcessingContext` to a concrete class?